src/libraries/Microsoft.Extensions.Http/src/Logging/LogHelper.cs

```diff
@@ -112,6 +127,11 @@ public static void LogRequestEnd(this ILogger logger, HttpResponseMessage respon
             }
         }
 
+        public static void LogRequestFailed(this ILogger logger, int statusCode, TimeSpan duration, string errorMessage)
+        {
+            _requestFailed(logger, duration.TotalMilliseconds, statusCode, errorMessage, null);
```

Comment:
We should pass the exception object to the logger to avoid losing useful information.
The exception message itself is often not useful (cancelled vs failed) compared to the stack trace and inner exceptions.

Reply:
Hey @MihaZupan ,
Updated